docs/user-guide/configure-origins.md

```diff
@@ -0,0 +1,166 @@
+# Configuring backend services and Origins
+
+## Configuration file
+Backend services and origins known to Styx are configured in their own YAML file, which is referenced by the main 
+configuration via the **originsFile** property.
+
+The value for this **originsFile** property accepts including environment variables (with the ${ENV_VAR} format) and can reference:
+ - A path in the filesystem by default.
+ - A resource in the classpath by using the prefix *classpath:*.
+
+###Examples:
+```yaml
+    originsFile=${STYX_HOME}/conf/env-development/origins.yaml
+    originsFile=classpath:/conf/origins.yaml
+ ```
+    
```

Comment:
This file is a great overall addition to our documentation. Thanks for putting it together.

However the above description about *originsFile* is slightly incomplete.

The origins file is actually specified in the `services/` section, under `backendServiceRegistry`, see: https://github.com/dvlato/styx/blob/5e18c02bdcf7e39e462f5c1232fe851fc3c1ed67/distribution/conf/default.yml#L59.

Note that this behaviour also depends on the advanced routing settings.

 The reason the above works is because of Styx configuration overrides.